core-bundle/tests/DependencyInjection/ContaoCoreExtensionTest.php

```diff
@@ -969,4 +969,11 @@ private function getContainerBuilder(array $params = null): ContainerBuilder
 
         return $container;
     }
+
+    private function skipTestIfAttributesAreNotSupported(): void
+    {
+        if (\PHP_VERSION_ID <= 80000) {
```

Comment:
```suggestion
        if (\PHP_VERSION_ID < 80000) {
```

PHP 8.0.0 should be supported I guess?

Reply:
Ah yes, my bad.